client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/step-content.tsx

```diff
@@ -64,7 +64,7 @@ const StepContent = ( { siteSlug, submit, goNext, goToStep, flow }: StepContentP
 					goNext={ goNext }
 					goToStep={ goToStep }
 					flow={ flow }
-					hideNavigation={ flow !== 'newsletter' }
+					hideNavigation={ true }
```

Comment:
Is this now `true` for all flows? If so, the whole mechanism and link could be removed from launchpad?

Reply:
I was a little nervous about removing it entirely since we've had a decent number of unintended consequences with different Stepper flows overlapping.

But it does seem like it's safe in this case.

Reply:
Yeah fair enough — it's important to get good at testing through them all. This one seems easy to confirm in code, too.